docusaurus/docs/how-to-guides/data-source-plugins/sql-requirements.md

```diff
@@ -0,0 +1,130 @@
+---
+id: sql-requirements
+title: Requirements to support SQL expressions
+sidebar_label: Support SQL expressions
+description: Data source requirements to support SQL expressions.
+keywords:
+  - grafana
+  - plugin
+  - datasource
+  - sql
+---
+
+Follow this guide to understand the plugin requirements to support SQL expressions.
+
+:::caution
+Only plugins with a backend can support SQL expressions.
+:::
+
+## Types of SQL responses
+
+Requirements depend on the type of response for your query. 
+
+SQL expressions support two types of responses:
+
+- [Tabular data response](#tabular-data-response) 
+- [Labeled metric time data response](#labeled-metric-time-data-response) 
+
+### Tabular data response
+
+A tabular data response is a single [data frame](../../key-concepts/data-frames) with no labels on any of the fields (columns). In short, a data frame that can be directly mapped to a SQL table.
+
+The supported types are:
```

Comment:
We don't really have a well defined type for table. So there there are no supported types to be listed here.